group_vars/model_zyxel_nwa55axe.yml

```diff
@@ -0,0 +1,18 @@
+---
+target: ramips/mt7621
+
+compat_version: 1.1
```

Comment:
why do you do this? what does this statement change?

Reply:
It's the same fix as here: https://github.com/freifunk-berlin/bbb-configs/commit/0eddf21ab7ed57491d63f38ec6db169f109e6f40

Switching from `dsa_ports` to `int_port` resulted in an upgrade warning.

Reply:
for me upgrading without a compat warning only works with compat_version: 1.1 and dsa_ports in combination. I will go with this one therefore.